src/Components/addItem.js

```diff
@@ -1,40 +1,91 @@
 import React from 'react';
+import '../App.css';
 import { fb } from '../lib/firebase';
 
 class AddItem extends React.Component {
   constructor() {
     super();
     this.state = {
       itemName: '',
+      purchaseFrequency: null,
+      lastPurchasedDate: null,
     };
   }
   updateInput = e => {
     this.setState({
       [e.target.name]: e.target.value,
     });
   };
+
+  updateFrequency = e => {
+    this.setState({
+      purchaseFrequency: e.target.value,
+    });
+  };
+  /* user is placeholder for user token when functionality is completed */
   addItem = e => {
     e.preventDefault();
     const db = fb.firestore();
-    db.collection('items').add({
+    db.collection('user').add({
       itemName: this.state.itemName,
+      purchaseFrequency: this.state.purchaseFrequency,
+      lastPurchasedDate: this.state.lastPurchasedDate,
     });
+
     this.setState({
       itemName: '',
+      purchaseFrequency: null,
+      lastPurchasedDate: null,
     });
```

Comment:
This works as-is but it took me a minute to figure out what this was doing. 
For maintainability and readability, this might benefit from being abstracted into a `resetInput` function/method.
The secondary benefit is that it splits up the tasks while your coding so you can isolate the 'reset' functionality from updating the database.